chart/values.schema.json

```
@@ -3580,7 +3580,7 @@
                         "query": {
                             "description": "Query to use for KEDA autoscaling. Must return a single integer.",
                             "type": "string",
-                            "default": "SELECT ceil(COUNT(*)::decimal / {{ .Values.config.triggerer.default_capacity }}) FROM trigger"
+                            "default": "SELECT ceil(COUNT(*)::decimal / {{ .Values.config.triggerer.capacity }}) FROM trigger"
```

```suggestion
                            "default": "SELECT ceil(COUNT(*)::decimal / {{ .Values.config.triggerer.capacity }}) FROM trigger"
```

As we chatted about, I think removing capacity from the chart config completely and doing something like this here makes more sense:

```
{{ .Values.config.triggerer.capacity | default .Values.config.triggerer.capacity | default 1000 }}
```

might be cleaner to have a new helper to dry this up too.